docs/decisions/0003-reduce-bokchoy-testing.rst

```diff
@@ -0,0 +1,58 @@
+Status
+~~~~~~
+Draft
+
+
+Context
+~~~~~~~
+
+edx-platform bokchoy tests are slow, flaky and difficult to debug.  A quick assessment of their value shows that they might be more trouble than they are worth.  And that we might get the same benefit with far fewer tests.
+
+Baseline Data:(Last 7 days)
+---------------------------
+
+    Total number of builds: 253(across 106 PRs)
+
+    Failures: 49(across 24 PRs)
+        True Failures: 10(across 6 PRs)
+        Failures that wouldn’t be caught by other test: 3(on 1 PR)
+
+Color
+=====
```

Comment:
Nit: I prefer stronger underlines (like `=`) for higher headings (e.g. Heading 1 rather than Heading 3), because when reading in raw format, it is more obvious that it is a main header.